Makefile.am

```diff
@@ -244,6 +265,7 @@ EXTRA_DIST += \
 	snapcraft/helpers \
 	snapcraft/snap \
 	babeld/Makefile \
+	mgmtd/Makefile \
```

Comment:
it isn't perfect sorting, but the daemons are sorted, I suggest putting this between `lib/Makefile` and `nhrpd/Makefile`

Reply:
Sure will take care of in the next force-push.